packages/core/admin/admin/src/content-manager/components/BlocksInput/BlocksEditor.tsx

```diff
@@ -38,8 +38,10 @@ interface BaseBlock {
   handleConvert?: (editor: Editor) => void | (() => React.JSX.Element);
   handleEnterKey?: (editor: Editor) => void;
   handleBackspaceKey?: (editor: Editor, event: React.KeyboardEvent<HTMLElement>) => void;
+  handleTab?: (editor: Editor) => void;
   snippets?: string[];
   dragHandleTopMargin?: CSSProperties['marginTop'];
+  styles?: string[];
```

Comment:
This is very specific to lists, could you avoid putting this in the store and move it, maybe in the list component?